cmake/03_compiler_config.cmake

```
@@ -171,6 +171,12 @@ if(ENABLE_LINTER)
   endif()
 endif()
 
+# Configure test coverage
+option(DISABLE_EXCEPTIONS "Disable exceptions and use CHECK instead" OFF)
+if(DISABLE_EXCEPTIONS)
+  add_definitions(-DDISABLE_EXCEPTIONS)
```

`add_definitions` has been deprecateed
https://cmake.org/cmake/help/latest/command/add_definitions.html
```suggestion
  add_compile_definitions(-DDISABLE_EXCEPTIONS)
```